compiler/rustc_codegen_ssa/src/back/symbol_export.rs

```diff
@@ -480,6 +477,17 @@ fn upstream_drop_glue_for_provider<'tcx>(
     }
 }
 
+fn upstream_async_drop_glue_for_provider<'tcx>(
+    tcx: TyCtxt<'tcx>,
+    args: GenericArgsRef<'tcx>,
+) -> Option<CrateNum> {
+    if let Some(def_id) = tcx.lang_items().async_drop_in_place_fn() {
```

Comment:
```suggestion
    let def_id = tcx.lang_items().async_drop_in_place_fn()?;
```

Reply:
Done in 1a8eae1aba998af1b0fdc597c0e263d4d391b169